<launch>
  <node name="detection" pkg="real_time_grasp" type="main.py" output="screen"/>
</launch>
